src/internal/observable/bindCallback.ts

```diff
@@ -4,48 +4,15 @@ import { bindCallbackInternals } from './bindCallbackInternals';
 
 // tslint:disable:max-line-length
 /** @deprecated resultSelector is no longer supported, use a mapping function. */
-export function bindCallback(callbackFunc: (...args: any[]) => any, resultSelector: (...args: any[]) => any, scheduler?: SchedulerLike): (...args: any[]) => Observable<any>;
+export function bindCallback(callbackFunc: (...args: any[]) => void, resultSelector: (...args: any[]) => void): (...args: any[]) => Observable<any>;
```

Comment:
issue: This change also remove the scheduler parameter. That seems wrong.

nitpick: The return type for `resultSelector` really should be `any`, as the internals use the return value. With `callbackFunc`, using `void` is fine, as we ignore any return value, but with `resultSelector` then return value should reflect its usage. A result selector that returns `void` makes no sense.

Reply:
Fixed